ibm_mq/tests/test_ibm_mq_unit.py

```diff
@@ -92,3 +92,20 @@ def test_channel_status_service_check_custom_mapping_invalid_config(aggregator,
 
     with pytest.raises(ConfigurationError):
         check.check(instance)
+
+
+@pytest.mark.parametrize('mqcd_version', [10, 'foo'])
```

Comment:
For even more clarity:

```suggestion
@pytest.mark.parametrize(
    'mqcd_version', [pytest.param(10, id='unsupported-version'), pytest.param('foo', id='not-an-int')]
)
```